tests/lib/rules/jsx-wrap-multilines.js

```diff
@@ -201,20 +232,6 @@ ruleTester.run('jsx-wrap-multilines', rule, {
       options: [{return: true}],
       errors: [{message: 'Missing parentheses around multilines JSX'}]
     }, {
-      code: DECLARATION_TERNARY_NO_PAREN,
```

Comment:
This is the same tests as tests for condition rule. I can return them if you think that they are useful. The more tests the better